llvm/include/llvm/Passes/StandardInstrumentations.h

```diff
@@ -46,6 +46,16 @@ class PrintIRInstrumentation {
   void registerCallbacks(PassInstrumentationCallbacks &PIC);
 
 private:
+  enum SuffixType {
+    before,
+    after,
+    invalidated,
+  };
```

Comment:
nit: I think this can be an `enum class` and all the types can be TitleCase (I think this is the prevailing style for new enums).